### ISSUE TITLE
Unexpected (and un-explained) panic when wails builds the HelloWorld example, despite wails doctor ok.

### ISSUE THREAD

**Issue by xavier268** [wailsapp/wails#3246]
### Description

Despite clean reinstall of latest golang, node, wails (v2) on a windows 11 (updated) machine, wails build panics (nil pointer) when attempting to build target.
Similar installation on another machine works perfectly.

### To Reproduce

install golang 1.22
install nvm-windows, select latest npm version (21.6.1)
install wails as per web site
run wails doctor -> ok
create hello world as per wails web site tutorial
run wails build in created folder --> crash

### Expected behaviour

Should run or at least **explain** what is causing the crash. 
There is obviously a configuration issue somewhere on my machine, since the same procedure works perfectly on another one.

### Attempted Fixes

tried to clean an reinstall go, nvm/node, remove entire go folder, remove path related to go,
Restart computer.
then reinstall everything ... but without success.

### System Details

```shell
((( wails doctor )))
Wails Doctor

# Wails
Version | v2.7.1

# System
┌──────────────────────────────────────────────────────────────────────────────────────┐
| OS           | Windows 10 Home                                                       |
| Version      | 2009 (Build: 22631)                                                   |
| ID           | 23H2                                                                  |
| Go Version   | go1.22.0                                                              |
| Platform     | windows                                                               |
| Architecture | amd64                                                                 |
| CPU          | Intel(R) Core(TM) i7-10700 CPU @ 2.90GHz                              |
| GPU 1        | NVIDIA GeForce RTX 2070 SUPER (NVIDIA) - Driver: 31.0.15.4609         |
| GPU 2        | Intel(R) UHD Graphics 630 (Intel Corporation) - Driver: 31.0.101.2125 |
| Memory       | 32GB                                                                  |
└──────────────────────────────────────────────────────────────────────────────────────┘

# Dependencies
┌───────────────────────────────────────────────────────┐
| Dependency | Package Name | Status    | Version       |
| WebView2   | N/A          | Installed | 121.0.2277.98 |
| Nodejs     | N/A          | Installed | 21.6.1        |
| npm        | N/A          | Installed | 10.2.4        |
| *upx       | N/A          | Available |               |
| *nsis      | N/A          | Available |               |
└─────────────── * - Optional Dependency ───────────────┘

# Diagnosis
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/

 SUCCESS  Your system is ready for Wails development!


((( wails build ))

Wails CLI v2.7.1


# Build Options

Platform(s)        | windows/amd64
Compiler           | C:\Program Files\Go\bin\go.exe
Skip Bindings      | false
Build Mode         | production
Devtools           | false
Frontend Directory | C:\Users\xavie\Desktop\helloworld\frontend
Obfuscated         | false
Skip Frontend      | false
Compress           | false
Package            | true
Clean Bin Dir      | false
LDFlags            |
Tags               | []
Race Detector      | false

# Building target: windows/amd64

panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x409f9e]

goroutine 25 [running]:
go/types.(*Checker).handleBailout(0xc0001e0800, 0xc0005d7bd0)
	C:/Program Files/Go/src/go/types/check.go:367 +0x88
panic({0xcd59c0?, 0x1a18b70?})
	C:/Program Files/Go/src/runtime/panic.go:770 +0x132
go/types.(*StdSizes).Sizeof(0x0, {0x1168db8, 0x1a262e0})
	C:/Program Files/Go/src/go/types/sizes.go:228 +0x31e
go/types.(*Config).sizeof(...)
	C:/Program Files/Go/src/go/types/sizes.go:333
go/types.representableConst.func1({0x1168db8?, 0x1a262e0?})
	C:/Program Files/Go/src/go/types/const.go:76 +0x9e
go/types.representableConst({0x116bfa0, 0xc0005b4228}, 0xc0001e0800, 0x1a262e0, 0xc0005d3e70)
	C:/Program Files/Go/src/go/types/const.go:92 +0x192
go/types.(*Checker).representation(0xc0001e0800, 0xc0003f1880, 0x1a262e0)
	C:/Program Files/Go/src/go/types/const.go:256 +0x65
go/types.(*Checker).implicitTypeAndValue(0xc0001e0800, 0xc0003f1880, {0x1168db8, 0x1a262e0})
	C:/Program Files/Go/src/go/types/expr.go:375 +0x30d
go/types.(*Checker).assignment(0xc0001e0800, 0xc0003f1880, {0x1168db8, 0x1a262e0}, {0xdeb6da, 0xe})
	C:/Program Files/Go/src/go/types/assignments.go:52 +0x2e5
go/types.(*Checker).exprInternal(0xc0001e0800, 0x0, 0xc0003f1880, {0x116a230, 0xc0003f0c40}, {0x0, 0x0})
	C:/Program Files/Go/src/go/types/expr.go:1175 +0x26dc
go/types.(*Checker).rawExpr(0xc0001e0800, 0x0, 0xc0003f1880, {0x116a230?, 0xc0003f0c40?}, {0x0?, 0x0?}, 0x0)
	C:/Program Files/Go/src/go/types/expr.go:979 +0x19e
go/types.(*Checker).expr(0xc0001e0800, 0x16ce29dd7b8?, 0xc0003f1880, {0x116a230?, 0xc0003f0c40?})
	C:/Program Files/Go/src/go/types/expr.go:1513 +0x30
go/types.(*Checker).unary(0xc0001e0800, 0xc0003f1880, 0xc00032c8a0)
	C:/Program Files/Go/src/go/types/expr.go:127 +0x4a
go/types.(*Checker).exprInternal(0xc0001e0800, 0x0, 0xc0003f1880, {0x116a830, 0xc00032c8a0}, {0x0, 0x0})
	C:/Program Files/Go/src/go/types/expr.go:1406 +0x1905
go/types.(*Checker).rawExpr(0xc0001e0800, 0x0, 0xc0003f1880, {0x116a830?, 0xc00032c8a0?}, {0x0?, 0x0?}, 0x1)
	C:/Program Files/Go/src/go/types/expr.go:979 +0x19e
go/types.(*Checker).genericExprList(0xc0001e0800, {0xc000214eb0, 0x1, 0xc0003f81b0?})
	C:/Program Files/Go/src/go/types/call.go:408 +0x385
go/types.(*Checker).callExpr(0xc0001e0800, 0xc0003f1840, 0xc0003f0c80)
	C:/Program Files/Go/src/go/types/call.go:303 +0x670
go/types.(*Checker).exprInternal(0xc0001e0800, 0x0, 0xc0003f1840, {0x116add0, 0xc0003f0c80}, {0x0, 0x0})
	C:/Program Files/Go/src/go/types/expr.go:1374 +0xf8
go/types.(*Checker).rawExpr(0xc0001e0800, 0x0, 0xc0003f1840, {0x116add0?, 0xc0003f0c80?}, {0x0?, 0x0?}, 0x0)
	C:/Program Files/Go/src/go/types/expr.go:979 +0x19e
go/types.(*Checker).multiExpr(0xc0001e0800, {0x116add0, 0xc0003f0c80}, 0x0)
	C:/Program Files/Go/src/go/types/expr.go:1532 +0x79
go/types.(*Checker).initVars(0xc0001e0800, {0xc000524280, 0x1, 0x1}, {0xc000214ec0, 0x1, 0x1}, {0x0, 0x0})
	C:/Program Files/Go/src/go/types/assignments.go:408 +0x129
go/types.(*Checker).shortVarDecl(0xc0001e0800, {0x11660e0, 0xc000818198}, {0xc000214e60, 0x1, 0x1?}, {0xc000214ec0, 0x1, 0x1})
	C:/Program Files/Go/src/go/types/assignments.go:556 +0x8f0
go/types.(*Checker).stmt(0xc0001e0800, 0x0, {0x116aa10, 0xc0003f0d40})
	C:/Program Files/Go/src/go/types/stmt.go:473 +0x15ff
go/types.(*Checker).stmtList(0xc0001e0800, 0x0, {0xc0003f1040?, 0x0?, 0x8?})
	C:/Program Files/Go/src/go/types/stmt.go:121 +0x85
go/types.(*Checker).funcBody(0xc0001e0800, 0x1168e08?, {0xc0005b40e4?, 0xc000432070?}, 0xc0003f1540, 0xc000123800, {0x0?, 0x0?})
	C:/Program Files/Go/src/go/types/stmt.go:41 +0x331
go/types.(*Checker).funcDecl.func1()
	C:/Program Files/Go/src/go/types/decl.go:852 +0x3a
go/types.(*Checker).processDelayed(0xc0001e0800, 0x0)
	C:/Program Files/Go/src/go/types/check.go:467 +0x162
go/types.(*Checker).checkFiles(0xc0001e0800, {0xc000214dc0, 0x2, 0x2})
	C:/Program Files/Go/src/go/types/check.go:411 +0x1cc
go/types.(*Checker).Files(...)
	C:/Program Files/Go/src/go/types/check.go:372
golang.org/x/tools/go/packages.(*loader).loadPackage(0xc0000ec2a0, 0xc000160810)
	C:/Users/xavie/go/pkg/mod/golang.org/x/tools@v0.6.0/go/packages/packages.go:1052 +0xa72
golang.org/x/tools/go/packages.(*loader).loadRecursive.func1()
	C:/Users/xavie/go/pkg/mod/golang.org/x/tools@v0.6.0/go/packages/packages.go:851 +0x1a9
sync.(*Once).doSlow(0xc00016ff98?, 0x129977?)
	C:/Program Files/Go/src/sync/once.go:74 +0xc2
sync.(*Once).Do(...)
	C:/Program Files/Go/src/sync/once.go:65
golang.org/x/tools/go/packages.(*loader).loadRecursive(0x432dfe?, 0xc00016ffd0?)
	C:/Users/xavie/go/pkg/mod/golang.org/x/tools@v0.6.0/go/packages/packages.go:839 +0x4a
golang.org/x/tools/go/packages.(*loader).refine.func2(0x0?)
	C:/Users/xavie/go/pkg/mod/golang.org/x/tools@v0.6.0/go/packages/packages.go:774 +0x26
created by golang.org/x/tools/go/packages.(*loader).refine in goroutine 1
	C:/Users/xavie/go/pkg/mod/golang.org/x/tools@v0.6.0/go/packages/packages.go:773 +0xccf

((( Actual system )))

Édition	Windows 11 Famille  <<<--- NOTE : wails doctor detects Windows 10 ?
Version	23H2
Installé le	‎05/‎10/‎2022
Build du système d’exploitation	22631.3085
Expérience	Windows Feature Experience Pack 1000.22684.1000.0
```

### Additional context

PS C:\Users\xavie\Desktop\helloworld> go mod graph

changeme github.com/bep/debounce@v1.2.1
changeme github.com/go-ole/go-ole@v1.2.6
changeme github.com/godbus/dbus/v5@v5.1.0
changeme github.com/google/uuid@v1.3.0
changeme github.com/jchv/go-winloader@v0.0.0-20210711035445-715c2860da7e
changeme github.com/labstack/echo/v4@v4.10.2
changeme github.com/labstack/gommon@v0.4.0
changeme github.com/leaanthony/go-ansi-parser@v1.6.0
changeme github.com/leaanthony/gosod@v1.0.3
changeme github.com/leaanthony/slicer@v1.6.0
changeme github.com/leaanthony/u@v1.1.0
changeme github.com/mattn/go-colorable@v0.1.13
changeme github.com/mattn/go-isatty@v0.0.19
changeme github.com/pkg/browser@v0.0.0-20210911075715-681adbf594b8
changeme github.com/pkg/errors@v0.9.1
changeme github.com/rivo/uniseg@v0.4.4
changeme github.com/samber/lo@v1.38.1
changeme github.com/tkrajina/go-reflector@v0.5.6
changeme github.com/valyala/bytebufferpool@v1.0.0
changeme github.com/valyala/fasttemplate@v1.2.2
changeme github.com/wailsapp/go-webview2@v1.0.10
changeme github.com/wailsapp/mimetype@v1.4.1
changeme github.com/wailsapp/wails/v2@v2.7.1
changeme go@1.21
changeme golang.org/x/crypto@v0.14.0
changeme golang.org/x/exp@v0.0.0-20230522175609-2e198f4a06a1
changeme golang.org/x/net@v0.17.0
changeme golang.org/x/sys@v0.13.0
changeme golang.org/x/text@v0.13.0
changeme toolchain@go1.22.0
github.com/go-ole/go-ole@v1.2.6 golang.org/x/sys@v0.0.0-20190916202348-b4ddaad3f8a3
github.com/jchv/go-winloader@v0.0.0-20210711035445-715c2860da7e golang.org/x/sys@v0.0.0-20200810151505-1b9f1253b3ed
github.com/labstack/echo/v4@v4.10.2 github.com/golang-jwt/jwt@v3.2.2+incompatible
github.com/labstack/echo/v4@v4.10.2 github.com/labstack/gommon@v0.4.0
github.com/labstack/echo/v4@v4.10.2 github.com/stretchr/testify@v1.8.1
github.com/labstack/echo/v4@v4.10.2 github.com/valyala/fasttemplate@v1.2.2
github.com/labstack/echo/v4@v4.10.2 golang.org/x/crypto@v0.6.0
github.com/labstack/echo/v4@v4.10.2 golang.org/x/net@v0.7.0
github.com/labstack/echo/v4@v4.10.2 golang.org/x/time@v0.3.0
github.com/labstack/echo/v4@v4.10.2 github.com/davecgh/go-spew@v1.1.1
github.com/labstack/echo/v4@v4.10.2 github.com/mattn/go-colorable@v0.1.13
github.com/labstack/echo/v4@v4.10.2 github.com/mattn/go-isatty@v0.0.17
github.com/labstack/echo/v4@v4.10.2 github.com/pmezard/go-difflib@v1.0.0
github.com/labstack/echo/v4@v4.10.2 github.com/valyala/bytebufferpool@v1.0.0
github.com/labstack/echo/v4@v4.10.2 golang.org/x/sys@v0.5.0
github.com/labstack/echo/v4@v4.10.2 golang.org/x/text@v0.7.0
github.com/labstack/echo/v4@v4.10.2 gopkg.in/yaml.v3@v3.0.1
github.com/labstack/gommon@v0.4.0 github.com/davecgh/go-spew@v1.1.1
github.com/labstack/gommon@v0.4.0 github.com/mattn/go-colorable@v0.1.11
github.com/labstack/gommon@v0.4.0 github.com/mattn/go-isatty@v0.0.14
github.com/labstack/gommon@v0.4.0 github.com/stretchr/testify@v1.7.0
github.com/labstack/gommon@v0.4.0 github.com/valyala/fasttemplate@v1.2.1
github.com/labstack/gommon@v0.4.0 golang.org/x/sys@v0.0.0-20211103235746-7861aae1554b
github.com/labstack/gommon@v0.4.0 gopkg.in/yaml.v3@v3.0.0-20210107192922-496545a6307b
github.com/leaanthony/go-ansi-parser@v1.6.0 github.com/matryer/is@v1.4.0
github.com/leaanthony/go-ansi-parser@v1.6.0 github.com/rivo/uniseg@v0.2.0
github.com/leaanthony/gosod@v1.0.3 github.com/leaanthony/debme@v1.2.1
github.com/leaanthony/gosod@v1.0.3 github.com/matryer/is@v1.4.0
github.com/leaanthony/slicer@v1.6.0 github.com/matryer/is@v1.4.0
github.com/leaanthony/u@v1.1.0 go@1.21
github.com/mattn/go-colorable@v0.1.13 github.com/mattn/go-isatty@v0.0.16
github.com/mattn/go-isatty@v0.0.19 golang.org/x/sys@v0.6.0
github.com/pkg/browser@v0.0.0-20210911075715-681adbf594b8 golang.org/x/sys@v0.0.0-20210616045830-e2b7044e8c71
github.com/samber/lo@v1.38.1 golang.org/x/exp@v0.0.0-20220303212507-bbda1eaf7a17
github.com/tkrajina/go-reflector@v0.5.6 github.com/stretchr/testify@v1.7.0
github.com/tkrajina/go-reflector@v0.5.6 github.com/davecgh/go-spew@v1.1.0
github.com/tkrajina/go-reflector@v0.5.6 github.com/pmezard/go-difflib@v1.0.0
github.com/tkrajina/go-reflector@v0.5.6 gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c
github.com/valyala/fasttemplate@v1.2.2 github.com/valyala/bytebufferpool@v1.0.0
github.com/wailsapp/go-webview2@v1.0.10 github.com/jchv/go-winloader@v0.0.0-20210711035445-715c2860da7e
github.com/wailsapp/go-webview2@v1.0.10 golang.org/x/sys@v0.8.0
github.com/wailsapp/mimetype@v1.4.1 golang.org/x/net@v0.0.0-20210505024714-0287a6fb4125
github.com/wailsapp/wails/v2@v2.7.1 github.com/Masterminds/semver@v1.5.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/acarl005/stripansi@v0.0.0-20180116102854-5a71ef0e047d
github.com/wailsapp/wails/v2@v2.7.1 github.com/bep/debounce@v1.2.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/bitfield/script@v0.19.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/charmbracelet/glamour@v0.5.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/flytam/filenamify@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/fsnotify/fsnotify@v1.4.9
github.com/wailsapp/wails/v2@v2.7.1 github.com/go-git/go-git/v5@v5.3.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/go-ole/go-ole@v1.2.6
github.com/wailsapp/wails/v2@v2.7.1 github.com/godbus/dbus/v5@v5.1.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/google/shlex@v0.0.0-20191202100458-e7afc7fbc510
github.com/wailsapp/wails/v2@v2.7.1 github.com/google/uuid@v1.3.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/jackmordaunt/icns@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/jaypipes/ghw@v0.12.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/labstack/echo/v4@v4.10.2
github.com/wailsapp/wails/v2@v2.7.1 github.com/labstack/gommon@v0.4.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/leaanthony/clir@v1.3.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/leaanthony/debme@v1.2.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/leaanthony/go-ansi-parser@v1.6.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/leaanthony/gosod@v1.0.3
github.com/wailsapp/wails/v2@v2.7.1 github.com/leaanthony/slicer@v1.6.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/leaanthony/u@v1.1.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/leaanthony/winicon@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/matryer/is@v1.4.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/pkg/browser@v0.0.0-20210911075715-681adbf594b8
github.com/wailsapp/wails/v2@v2.7.1 github.com/pkg/errors@v0.9.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/pterm/pterm@v0.12.49
github.com/wailsapp/wails/v2@v2.7.1 github.com/sabhiram/go-gitignore@v0.0.0-20210923224102-525f6e181f06
github.com/wailsapp/wails/v2@v2.7.1 github.com/samber/lo@v1.38.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/stretchr/testify@v1.8.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/tc-hib/winres@v0.2.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/tidwall/sjson@v1.1.7
github.com/wailsapp/wails/v2@v2.7.1 github.com/tkrajina/go-reflector@v0.5.6
github.com/wailsapp/wails/v2@v2.7.1 github.com/wailsapp/go-webview2@v1.0.10
github.com/wailsapp/wails/v2@v2.7.1 github.com/wailsapp/mimetype@v1.4.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/wzshiming/ctc@v1.2.3
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/mod@v0.12.0
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/net@v0.17.0
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/sys@v0.13.0
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/tools@v0.6.0
github.com/wailsapp/wails/v2@v2.7.1 atomicgo.dev/cursor@v0.1.1
github.com/wailsapp/wails/v2@v2.7.1 atomicgo.dev/keyboard@v0.2.8
github.com/wailsapp/wails/v2@v2.7.1 bitbucket.org/creachadair/shell@v0.0.7
github.com/wailsapp/wails/v2@v2.7.1 github.com/Microsoft/go-winio@v0.4.16
github.com/wailsapp/wails/v2@v2.7.1 github.com/StackExchange/wmi@v1.2.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/alecthomas/chroma@v0.10.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/aymerick/douceur@v0.2.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/containerd/console@v1.0.3
github.com/wailsapp/wails/v2@v2.7.1 github.com/davecgh/go-spew@v1.1.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/dlclark/regexp2@v1.4.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/emirpasic/gods@v1.12.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/ghodss/yaml@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/go-git/gcfg@v1.5.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/go-git/go-billy/v5@v5.2.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/gookit/color@v1.5.2
github.com/wailsapp/wails/v2@v2.7.1 github.com/gorilla/css@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/imdario/mergo@v0.3.12
github.com/wailsapp/wails/v2@v2.7.1 github.com/jaypipes/pcidb@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/jbenet/go-context@v0.0.0-20150711004518-d14ea06fba99
github.com/wailsapp/wails/v2@v2.7.1 github.com/jchv/go-winloader@v0.0.0-20210711035445-715c2860da7e
github.com/wailsapp/wails/v2@v2.7.1 github.com/kevinburke/ssh_config@v0.0.0-20201106050909-4977a11b4351
github.com/wailsapp/wails/v2@v2.7.1 github.com/kr/pretty@v0.3.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/lithammer/fuzzysearch@v1.1.5
github.com/wailsapp/wails/v2@v2.7.1 github.com/lucasb-eyer/go-colorful@v1.2.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/mattn/go-colorable@v0.1.13
github.com/wailsapp/wails/v2@v2.7.1 github.com/mattn/go-isatty@v0.0.19
github.com/wailsapp/wails/v2@v2.7.1 github.com/mattn/go-runewidth@v0.0.13
github.com/wailsapp/wails/v2@v2.7.1 github.com/microcosm-cc/bluemonday@v1.0.17
github.com/wailsapp/wails/v2@v2.7.1 github.com/mitchellh/go-homedir@v1.1.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/muesli/reflow@v0.3.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/muesli/termenv@v0.9.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/nfnt/resize@v0.0.0-20180221191011-83c6a9932646
github.com/wailsapp/wails/v2@v2.7.1 github.com/olekukonko/tablewriter@v0.0.5
github.com/wailsapp/wails/v2@v2.7.1 github.com/pmezard/go-difflib@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/rivo/uniseg@v0.4.4
github.com/wailsapp/wails/v2@v2.7.1 github.com/sergi/go-diff@v1.2.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/tidwall/gjson@v1.9.3
github.com/wailsapp/wails/v2@v2.7.1 github.com/tidwall/match@v1.1.1
github.com/wailsapp/wails/v2@v2.7.1 github.com/tidwall/pretty@v1.2.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/valyala/bytebufferpool@v1.0.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/valyala/fasttemplate@v1.2.2
github.com/wailsapp/wails/v2@v2.7.1 github.com/wzshiming/winseq@v0.0.0-20200112104235-db357dc107ae
github.com/wailsapp/wails/v2@v2.7.1 github.com/xanzy/ssh-agent@v0.3.0
github.com/wailsapp/wails/v2@v2.7.1 github.com/xo/terminfo@v0.0.0-20210125001918-ca9a967f8778
github.com/wailsapp/wails/v2@v2.7.1 github.com/yuin/goldmark@v1.4.13
github.com/wailsapp/wails/v2@v2.7.1 github.com/yuin/goldmark-emoji@v1.0.1
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/crypto@v0.14.0
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/exp@v0.0.0-20230522175609-2e198f4a06a1
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/image@v0.12.0
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/term@v0.13.0
github.com/wailsapp/wails/v2@v2.7.1 golang.org/x/text@v0.13.0
github.com/wailsapp/wails/v2@v2.7.1 gopkg.in/warnings.v0@v0.1.2
github.com/wailsapp/wails/v2@v2.7.1 gopkg.in/yaml.v2@v2.4.0
github.com/wailsapp/wails/v2@v2.7.1 gopkg.in/yaml.v3@v3.0.1
github.com/wailsapp/wails/v2@v2.7.1 howett.net/plist@v1.0.0
go@1.21 toolchain@go1.21
golang.org/x/crypto@v0.14.0 golang.org/x/net@v0.10.0
golang.org/x/crypto@v0.14.0 golang.org/x/sys@v0.13.0
golang.org/x/crypto@v0.14.0 golang.org/x/term@v0.13.0
golang.org/x/crypto@v0.14.0 golang.org/x/text@v0.13.0
golang.org/x/exp@v0.0.0-20230522175609-2e198f4a06a1 github.com/google/go-cmp@v0.5.8
golang.org/x/exp@v0.0.0-20230522175609-2e198f4a06a1 golang.org/x/mod@v0.6.0
golang.org/x/exp@v0.0.0-20230522175609-2e198f4a06a1 golang.org/x/tools@v0.2.0
golang.org/x/exp@v0.0.0-20230522175609-2e198f4a06a1 golang.org/x/sys@v0.1.0
golang.org/x/net@v0.17.0 golang.org/x/crypto@v0.14.0
golang.org/x/net@v0.17.0 golang.org/x/sys@v0.13.0
golang.org/x/net@v0.17.0 golang.org/x/term@v0.13.0
golang.org/x/net@v0.17.0 golang.org/x/text@v0.13.0
golang.org/x/text@v0.13.0 golang.org/x/tools@v0.6.0
golang.org/x/text@v0.13.0 golang.org/x/mod@v0.8.0
golang.org/x/text@v0.13.0 golang.org/x/sys@v0.5.0
github.com/mattn/go-colorable@v0.1.11 github.com/mattn/go-isatty@v0.0.14
github.com/mattn/go-colorable@v0.1.11 golang.org/x/sys@v0.0.0-20210927094055-39ccf1dd6fa6
github.com/mattn/go-isatty@v0.0.14 golang.org/x/sys@v0.0.0-20210630005230-0f9fa26af87c
github.com/stretchr/testify@v1.7.0 github.com/davecgh/go-spew@v1.1.0
github.com/stretchr/testify@v1.7.0 github.com/pmezard/go-difflib@v1.0.0
github.com/stretchr/testify@v1.7.0 github.com/stretchr/objx@v0.1.0
github.com/stretchr/testify@v1.7.0 gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c
github.com/valyala/fasttemplate@v1.2.1 github.com/valyala/bytebufferpool@v1.0.0
gopkg.in/yaml.v3@v3.0.0-20210107192922-496545a6307b gopkg.in/check.v1@v0.0.0-20161208181325-20d25e280405
github.com/leaanthony/debme@v1.2.1 github.com/leaanthony/slicer@v1.5.0
github.com/leaanthony/debme@v1.2.1 github.com/matryer/is@v1.4.0
github.com/mattn/go-isatty@v0.0.16 golang.org/x/sys@v0.0.0-20220811171246-fbc7d0a398ab
gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c gopkg.in/check.v1@v0.0.0-20161208181325-20d25e280405
golang.org/x/net@v0.0.0-20210505024714-0287a6fb4125 golang.org/x/sys@v0.0.0-20210423082822-04245dca01da
golang.org/x/net@v0.0.0-20210505024714-0287a6fb4125 golang.org/x/term@v0.0.0-20201126162022-7de9c90e9dd1
golang.org/x/net@v0.0.0-20210505024714-0287a6fb4125 golang.org/x/text@v0.3.6
golang.org/x/term@v0.0.0-20201126162022-7de9c90e9dd1 golang.org/x/sys@v0.0.0-20201119102817-f84b799fce68
golang.org/x/text@v0.3.6 golang.org/x/tools@v0.0.0-20180917221912-90fa682c2a6e

**Comment by stffabi** [comment]
Thanks for using Wails and reporting this issue.

This issue has already been fixed on master and is a duplicate of #3242.

**Comment by stffabi** [comment]
Duplicate of #3242

**Comment by xavier268** [comment]
Thanks for taking the time to answer - I will look forward to the next
release 😊

Xavier

**Comment by leaanthony** [comment]
Fixed in release v2.8.0

**Comment by xavier268** [comment]
Confirmed as solved. Thanks to the team for their reactivity ! Xavier
